simplesat/sat/policy.py

```diff
@@ -59,7 +59,11 @@ def _log_pretty_pkg_id(self, pkg_id):
         package = self._log_pool._id_to_package[pkg_id]
         name_ver = '{} {}'.format(package.name, package.version)
         fill = '.' if pkg_id % 2 else ''
-        return "{:{fill}<30} {}".format(name_ver, pkg_id, fill=fill)
+        try:
+            repo = package.repository_info.name
```

Comment:
@cournape, the attribute error is because `InstalledPackageMetadata` has no `repository_info`.
```
Traceback (most recent call last):
  File "/home/john/projects/enthought/jaguar/.devenv-jag34/bin/jaguar", line 9, in <module>
    load_entry_point('jaguar', 'console_scripts', 'jaguar')()
...
...
  File "/home/john/projects/enthought/sat-solver/simplesat/sat/policy.py", line 76, in _log_pretty_pkg_id
    repo = package.repository_info.name
AttributeError: 'InstalledPackageMetadata' object has no attribute 'repository_info'

```